ingestion/src/metadata/ingestion/source/dashboard/quicksight/metadata.py

```diff
@@ -161,36 +170,37 @@ def yield_dashboard_chart(
         # Each dashboard is guaranteed to have at least one sheet, which represents
         # a chart in the context of QuickSight
         # https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/quicksight.html#QuickSight.Client.describe_dashboard
-        charts = dashboard_details["Version"]["Sheets"]
-        for chart in charts:
-            try:
-                if filter_by_chart(
-                    self.source_config.chartFilterPattern, chart["Name"]
-                ):
-                    self.status.filter(chart["Name"], "Chart Pattern not allowed")
-                    continue
+        if dashboard_details.Version:
+            charts = dashboard_details.Version.Sheets
+            for chart in charts:
```

Comment:
I believe you should be able to do directly ` for sheet in dashboard_details.Version.Sheets`. No need to create the `charts` var.

nit: We should probably rename `chart` to `sheet` since we are accessing `.Sheets`.

Reply:
Here in `quicksight`, the `sheets` represents `charts`.

Reply:
Removed the charts variable iterating over directly.

Reply:
in the pydantic Models, we can do that, it will make more sense then.

Reply:
Renamed sheets to charts in models.